tf_files/aws/kube.tf

```diff
@@ -35,18 +35,18 @@ resource "aws_subnet" "public_kube" {
 }
 
 
-resource "aws_db_instance" "db_userapi" {
+resource "aws_db_instance" "db_fence" {
```

Comment:
We should try to make our cloud-formation changes backward compatible, so we can easily apply them to existing commons, and deploy fence without undeploying user-api, so we can have an easy rollback.
Rather than remove the 'db_userapi' RDS instance -leave that in place, and add a new 'db_fence' instance, so we can run the 2 side by side for a while.  At the point that a commons is completely transitioned over to fence, then we can remove the userapi db later by setting a 'userapi_count' varialbe that defaults to 1 to 0:
     https://blog.gruntwork.io/terraform-tips-tricks-loops-if-statements-and-gotchas-f739bbae55f9